cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/SteadyStateHypothesisExport.java

```diff
@@ -443,14 +460,17 @@ private static void addRegulatingControlView(TapChanger<?, ?, ?, ?> tc, CgmesTap
             RegulatingControlView rcv = null;
             if (tc instanceof RatioTapChanger ratioTapChanger
                     && CgmesExportUtil.regulatingControlIsDefined(ratioTapChanger)) {
+                String unitMultiplier = switch (ratioTapChanger.getRegulationMode()) {
+                    case VOLTAGE -> "k";
+                    case REACTIVE_POWER -> "M";
+                };
                 rcv = new RegulatingControlView(controlId,
```

Comment:
Maybe use the method used in the EQ export to obtain the mode and then depending on the mode set the unitMultiplier